package com.obsqura.Training.Exceptions;

public class ArithameticExc {

	public static void main(String[] args) {
	
		try
		{
		int x=70/0;	
		System.out.println("DIVISION IS SUCCESSFUL");
			
		}
		catch(ArithmeticException e)
		{
			
		System.out.println("YOU CAN'T DIVIDE A NUMBER BY ZERO");	
			
			
		}
		
		
		
		
		
		
		
		

	}

}
